From 2d0c2cade5da2060c276f4173aa2253f460b69d3 Mon Sep 17 00:00:00 2001
From: Cristian Gafton <gafton@amazon.com>
Date: Wed, 13 Feb 2013 05:34:40 +0000
Subject: bump default tcp_wmem from 16KB to 20KB

to account for skb structure growth since the 3.4.x kernel series

(cherry picked from commit 4859c886f6ee760a0d99a68ee7711026090a1746)
Signed-off-by: Cristian Gafton <gafton@amazon.com>
(cherry picked from commit 64f08e9da01ff957cbebd44de8797373c589545f)
---
 net/ipv4/tcp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/ipv4/tcp.c b/net/ipv4/tcp.c
index 036a76b..df65d88 100644
--- a/net/ipv4/tcp.c
+++ b/net/ipv4/tcp.c
@@ -3177,7 +3177,7 @@ void __init tcp_init(void)
 	max_rshare = min(6UL*1024*1024, limit);
 
 	sysctl_tcp_wmem[0] = SK_MEM_QUANTUM;
-	sysctl_tcp_wmem[1] = 16*1024;
+	sysctl_tcp_wmem[1] = 20*1024;
 	sysctl_tcp_wmem[2] = max(64*1024, max_wshare);
 
 	sysctl_tcp_rmem[0] = SK_MEM_QUANTUM;
-- 
2.7.4

